app/src/services/clientStorageService/hooks/useClientStorageService.ts

```diff
@@ -1,14 +1,22 @@
-import { useEffect } from "react";
+import Logger from "lib/logger";
 import { useSelector } from "react-redux";
 import { getAppMode } from "store/selectors";
 import { initClientStorageService } from "..";
-import Logger from "lib/logger";
+import { useEffect } from "react";
+
+let firstInit = false;
```

Comment:
please change this to ref if possible. I'm afraid this might get cleaned up by the GC.

Reply:
Done